lib/ret/login.ex

```diff
@@ -22,4 +24,33 @@ defmodule Ret.Login do
     |> unique_constraint(:identifier_hash)
     |> put_assoc(:account, account)
   end
+
+  def update_identifier_hash(%{old_email: old_email, new_email: new_email} = _) do
+    old_identifier_hash = Account.identifier_hash_for_email(old_email)
+    new_identifier_hash = Account.identifier_hash_for_email(new_email)
+    login = Repo.get_by(Login, identifier_hash: old_identifier_hash)
+    case update_identifier_hash_for_login(login, new_identifier_hash) do
+      {:error, :no_login} -> {:error, :no_account_for_old_email}
+      {:error, :new_identifier_hash_is_not_unique} -> {:error, :new_email_already_in_use}
+      {:error, _} -> {:error, :failed_to_update_login}
```

Comment:
Ditto

Reply:
Resolved in https://github.com/mozilla/reticulum/commit/eee56865456bf34ae00958478c1ed8e15a49007b